continuous_integration/gpuci/build.sh

```diff
@@ -37,6 +37,17 @@ gpuci_logger "Activate conda env"
 . /opt/conda/etc/profile.d/conda.sh
 conda activate dask_sql
 
+gpuci_logger "Install awscli"
+gpuci_mamba_retry install -y -c conda-forge awscli
+
+gpuci_logger "Download TPC-DS dataset"
+gpuci_retry aws s3 cp --only-show-errors s3://rapidsai-data/tpcx-bb-data/tpc-ds/sf1/parquet_2gb/ tests/unit/data/ --recursive
+
+gpuci_logger "Pull in query files"
+git clone https://github.com/databricks/spark-sql-perf.git
+mkdir -p tests/unit/queries
+cp spark-sql-perf/src/main/resources/tpcds_2_4/* tests/unit/queries
```

Comment:
Just pointing out that this would probably make more sense to do during the image builds versus on a per-run basis; once this is in can refactor to make that the case